client/src/types.d.ts

```diff
@@ -362,10 +362,10 @@ export type Task = {
 // User
 export type User = {
   displayName: string;
```

Comment:
`displayName` will be removed in the API

Reply:
I will remove this property and replace it on components by the user full name